bond.sh

```diff
@@ -1,41 +1,57 @@
 #!/bin/bash
 
-# Coming to Delta-11, 'not tested yet'
-
-# Propose validator bid.
+# Tested and works on "delta-10"
 
+# Bond validat to networks
 # Requirements: 'apt install jq'
-# Requirements: 'add_bid.wasm' should be compiled according instruction. Current 'Delta 11' target 'v0.9.3'
-# Requirements: 'BID AMOUNT', 'validator comission', 'CHAIN NAME', 'OWNER PRIVATE KEY PATH', 'TARGET IP', 'ADD BID WASM PATH'.
+# Requirements: Set 'validator public hex' , 'BID_AMOUNT' , 'PROFIT ( fee ), 'CHAIN_NAME', 'OWNER_PRIVATE_KEY' path, 'API' end pint, 'BONDING_CONTRACT' path.
 
-# Note the path to files and keys. Note: the 'session arguments need to be encased in double-quotes', with the 'parameter values in single quotes'.
-# Note the required payment amount. It must contain 'at least 10 zeros'. The payment amount is specified 'in motes'.
+PUB_KEY_HEX="$1"
 
-PUB_KEY_HEX="$(cat /etc/casper/validator_keys/public_key_hex)"
-BID_AMOUNT="1234567890"
+BID_AMOUNT="978000000000"
+GAS="1000000000" # So far this is minimum which I be able to achive, 10 zeros
 
-payment_amount="10000000000"
+PROFIT="10"
 
-validator_comission="10"
-CHAIN_NAME="delta-11"
+CHAIN_NAME=`curl -s localhost:8888/status | jq -r .chainspec_name`
```

Comment:
If call been made, we need to add checks for bogus data if any, for example service is down.